### PR TITLE
Internal UI - Displaying user messages

### PR DESCRIPTION
**Pull request by justinorringer**
messages

### CODE REVIEW

File: internal-ui/src/pages/UserMessagesPage.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import {MessageUser, Session} from "../Types.ts";
+import {useMessages} from "../services/messages.ts";
+import {LoadingPageSection} from "../components/LoadingPageSection.tsx";
+import { UserMessages } from "../components/UserMessages.tsx";
+
+
+export const UserMessagesPage = () => {
+
+    const messagesQuery = useMessages();
```
Comment (josejulio) [inline internal-ui/src/pages/UserMessagesPage.tsx:16]: It is worth to add a param here (or options variable) to be able to set the message type.
I see that you are using `user`.

It's currently sending these https://github.com/RedHatInsights/astro-virtual-assistant/blob/main/internal-ui/src/services/messages.ts#L5-L9 (which is that could be the default if this new param is unset)

Comment (justinorringer) [reply internal-ui/src/pages/UserMessagesPage.tsx:16]: To see how many times the bot says something could be useful, makes sense :thinking:

**Comment by justinorringer** [comment]
ui

**Review by tahmidefaz — APPROVED** [review]
Looks good!
Question: Are the changes under `internal/public/index.html` auto generated? Kind of feels odd that the .css and .js files under `internal/public/assets` changes whenever we incorporate new things.

**Comment by justinorringer** [comment]
> Looks good! Question: Are the changes under `internal/public/index.html` auto generated? Kind of feels odd that the .css and .js files under `internal/public/assets` changes whenever we incorporate new things.

Yeah they are. not sure why we are building it like that though

**Comment by josejulio** [comment]
> Yeah they are. not sure why we are building it like that though

They are built like that to spare us from having to build them on docker build - I don't remember the exact details that lead to that, but not opposed to move it to docker build time.

**Pull request merged**